manifests/config.pp

```diff
@@ -40,15 +40,7 @@
     }
   }
 
-  $_config.each |$setting, $value| {
-    ini_setting { "${config_file} ${setting} ${value}":
-      ensure  => present,
-      path    => $config_file,
-      section => '',
-      setting => $setting,
-      value   => $value,
-      require => File['/etc/letsencrypt'],
-    }
-  }
+  $_config_joined = join_keys_to_values($_config, '=')
+  letsencrypt::config::ini { $_config_joined: }
```

Comment:
As discussed on IRC, this was PUP-4709.  I sent inifile a PR at https://github.com/puppetlabs/puppetlabs-inifile/pull/195 to remove the square brackets, but for now I've restored the letsencrypt::config::ini define.